### ISSUE TITLE
error message

### ISSUE THREAD

**Issue by anvaruk** [BoingBoing/slack-analytics#1]
when I hit dailydigest.php I get error message in browser

There was a general error : Error refreshing the OAuth2 token, message: '{ "error" : "unauthorized_client", "error_description" : "Unauthorized client or scope in request." }'

I'm sure I set up correctly 
'GOOGLE_CLIENT_EMAIL'  and 'GOOGLE_USER_EMAIL'
and also generated P12 key and uploded to the same folder

**Comment by evansobkowicz** [comment]
I'm having the same issue... Do you have any other configuration instructions?

**Comment by funcoding** [comment]
Even I am facing same issue. Has anyone found a solution?

**Comment by deadrich** [comment]
I have forked the repo and created a fix for this error.  I have submitted pull request, but if you want to get the changes right away they are at https://github.com/deadrich/slack-analytics

**Comment by anvaruk** [comment]
thanks
but slack has not worked out for us, we are now using telegram messenger, which is much simpler and more convenient for users, it also has API for 3rd party development and ready bots

**Comment by evansobkowicz** [comment]
@deadrich Thanks! That was really helpful. I also had to remove the sub email address in order to make my auth error go away... This Line: https://github.com/BoingBoing/slack-analytics/blob/master/setup.example.php#L55

**Comment by deadrich** [comment]
@anvaruk Are you using Telegram Messenger with Slack or as a complete Slack replacement?

**Comment by deadrich** [comment]
@evansobkowicz Good catch!  I had to remove that line as well.  I have committed the change.

**Comment by anvaruk** [comment]
@deadrich complete replacement
its difficult to force users so they found best suitable system for them)
regarding telegram bots I suggest trying @Jaconda , its has got web interface for managing integrations, and we created own bot based on https://github.com/yagop/telegram-bot to fetch stats from google analytics

**Comment by deanputney** [comment]
Uh, hello! I forgot entirely that I made this repo. I'm no longer with Boing Boing (now at [Glowforge](http://glowforge.com)) but I'm glad you guys are interested in this! I'll review issues and pull requests soon!
